packages/backend/src/ee/services/ScimService/ScimService.ts

```diff
@@ -1143,7 +1151,7 @@ export class ScimService extends BaseService {
                 oldName: existingGroup.name,
                 newName: patchedScimGroup.displayName,
                 oldMemberCount: existingGroup.members.length,
-                newMemberCount: patchedScimGroup.members.length,
+                newMemberCount: (patchedScimGroup.members ?? []).length,
```

Comment:
We assume that tsoa zod validation throws if `patchedScimGroup` is not `undefined` or an array?